src/Drupal/Commands/core/BaseFieldOverrideCreateCommands.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Drush\Drupal\Commands\core;
+
+use Drupal\Core\Entity\EntityFieldManager;
+use Drupal\Core\Entity\EntityTypeBundleInfo;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\Entity\BaseFieldOverride;
+use Drush\Commands\DrushCommands;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class BaseFieldOverrideCreateCommands extends DrushCommands
+{
+    use AskBundleTrait;
+    use ValidateEntityTypeTrait;
+
+    /** @var EntityTypeManagerInterface */
+    protected $entityTypeManager;
+    /** @var EntityTypeBundleInfo */
+    protected $entityTypeBundleInfo;
+    /** @var EntityFieldManager */
+    protected $entityFieldManager;
+
+    public function __construct(
+        EntityTypeManagerInterface $entityTypeManager,
+        EntityTypeBundleInfo $entityTypeBundleInfo,
+        EntityFieldManager $entityFieldManager
+    ) {
+        $this->entityTypeManager = $entityTypeManager;
+        $this->entityTypeBundleInfo = $entityTypeBundleInfo;
+        $this->entityFieldManager = $entityFieldManager;
+    }
+
+    /**
+     * Create a new base field override
+     *
+     * @command base-field:override-create
```

Comment:
I think we want this in the field: namespace so it groups with the others in help list.

Reply:
Changed